staff/joseluis-juste/cv/Index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Currículum Vitae</title>
+</head>
+
+<body>
+        <fieldset>
```

Comment:
fieldsets are part of a form, not part of the main content of a web, please RTFM https://www.w3schools.com/tags/tag_fieldset.asp